test/test_export_vm.py

```diff
@@ -96,3 +99,17 @@ def test_export_vm(aws_vm_export_mock, default_asset_id, vm_formats_to_test, tes
              s3_prefix=f"{default_asset_id.bucket_prefix}/")
         for disk_format in vm_formats_to_test]
     assert mock_cast(aws_vm_export_mock.export_ami_image_to_vm).call_args_list == expected_calls
+
+    expected_calls_copy = list()
+    expected_calls_delete = list()
+    for disk_format in vm_formats_to_test:
+        source = build_image_source(prefix=default_asset_id.bucket_prefix,
+                                    export_image_task_id="export-ami-123",
+                                    vm_image_format=disk_format)
+        dest = build_image_destination(prefix=default_asset_id.bucket_prefix, asset_id=default_asset_id,
+                                       vm_image_format=disk_format)
+        expected_calls_copy.append(call(bucket=TEST_BUCKET_ID, source=source, dest=dest))
+        expected_calls_delete.append(call(bucket=TEST_BUCKET_ID, source=source))
+
+    assert mock_cast(aws_vm_export_mock.copy_s3_object).call_args_list == expected_calls_copy
```

Comment:
Is there are a reason you are using call_args_list

Reply:
From what I remember that was the way to validate multiple calls to a function with specific arguments

Reply:
ok, many ways to get to same, I usually use [mock_calls](https://docs.python.org/3/library/unittest.mock.html#unittest.mock.Mock.mock_calls) or [assert_has_calls](https://docs.python.org/3/library/unittest.mock.html#unittest.mock.Mock.assert_has_calls)